lib/std/fmt.zig

```diff
@@ -2059,6 +2059,8 @@ test "comptimePrint" {
     @setEvalBranchQuota(2000);
     try std.testing.expectEqual(*const [3:0]u8, @TypeOf(comptimePrint("{}", .{100})));
     try std.testing.expectEqualSlices(u8, "100", comptimePrint("{}", .{100}));
+    try std.testing.expectComptimeFmt("30", "{d}", .{30.0});
+    try std.testing.expectComptimeFmt("30.0", "{d:3.1}", .{30.0});
```

Comment:
I believe this should be equivalent without the need for the new `expectComptimeFmt` function:
```suggestion
    try std.testing.expectEqualStrings("30", comptimePrint("{d}", .{30.0}));
    try std.testing.expectEqualStrings("30.0", comptimePrint("{d:3.1}", .{30.0}));
```

Reply:
Thank you @squeek502 for pointing this out. I'll update my pull request.

Reply:
(@squeek502  uhm.. I just realized I could simply merge your commit, right? Sorry, this is my very first pull request)

Reply:
No worries, what you did is good.